exporters/otlp/common/src/test/java/io/opentelemetry/exporter/internal/otlp/traces/TraceRequestMarshalerTest.java

```diff
@@ -123,13 +138,14 @@ void toProtoSpan() {
                         Collections.singletonList(
                             EventData.create(12347, "my_event", Attributes.empty())))
                     .setTotalRecordedEvents(3)
-                    .setLinks(Collections.singletonList(LinkData.create(SPAN_CONTEXT)))
+                    .setLinks(Collections.singletonList(LinkData.create(spanContext)))
                     .setTotalRecordedLinks(2)
                     .setStatus(StatusData.ok())
                     .build()));
 
     assertThat(span.getTraceId().toByteArray()).isEqualTo(TRACE_ID_BYTES);
     assertThat(span.getSpanId().toByteArray()).isEqualTo(SPAN_ID_BYTES);
+    assertThat(span.getTraceState()).isEqualTo(encodeTraceState(spanContext.getTraceState()));
```

Comment:
Let's put the actual trace state string inline, e.g. `"foo=bar,baz=qux".getBytes(UTF_8)"` so it's  more obvious the test is working